theme/styles/base/_base.scss

```diff
@@ -3,16 +3,6 @@
 @use "abstracts/mixins/base";
 @use "abstracts/mixins/layout";
 
-// Base
```

Comment:
CSS loading order got a little mixed up and these styles that were previously overwritten surfaced. The box-sizing is unnecessary since we already have a reset that handles it, and the `[loading]` is outdated and ends up hiding the images since most image elements have a loading attribute used to specific if they are lazy or eager